### PR TITLE
Drawer

### PR DESCRIPTION
**Pull request by NunoSempere**
Adds a distribution drawer to widedomain.

Things to note:
- The code has comments. I feel protective of these comments and feel that they help structure the code and will help me out when I come I come back to this code a couple of weeks or months from now.
- Originally based on code by Evan Ward (probability.dev), and heavily inspired by it. See also: observablehq.com/@nunosempere/distribution-drawer

Cool things yet to be done:
- Add other features from probability.dev
- Make it so that one can input a guesstimate function, and then draw on it. To do this, use the Convert.xyShapeToCanvasShape and modify the Draw.initial distribution function slightly.
- Maybe reach out to Metaculus to see if they want to use this somewhere?

### CODE REVIEW

**Comment by NunoSempere** [comment]
Any idea why those checks are falling / what I would need to do to fix them?

**Comment by OAGr** [comment]
I'm really not sure. I tried rebuilding it on Netlify but no dice yet. These things can be tricky to debug. I've had mixed success with Netlify and ReasonML, it's frustrating for sure.

Deployment in general can be quite tricky.

File: package.json [hunk 1]
```diff
@@ -36,9 +36,11 @@
     "binary-search-tree": "0.2.6",
     "bs-ant-design-alt": "2.0.0-alpha.33",
     "bs-css": "11.0.0",
+    "bs-fetch": "^0.5.2",
     "bs-moment": "0.4.4",
-    "bs-platform": "7.2.2",
+    "bs-platform": "^7.2.2",
```
Comment (OAGr) [inline package.json:41]: why did you add this? The bucklescript version can be a bit fickle.

Comment (NunoSempere) [reply package.json:41]: I don't remember; doesn't seem to be necessary. Will keep bucklescript's fickleness in mind.
